Standards/scs-0100-w1-flavor-naming-implementation-testing.md

```diff
@@ -45,6 +46,107 @@ will create a whole set of flavors in one go.
 To that end, it provides different options: either the standard mandatory and
 possibly recommended flavors can be created, or the user can set a file containing his flavors.
 
+### GPU table
+
+The most commonly used datacenter GPUs are listed here, showing what GPUs (or partitions
+of a GPU) result in what GPU part of the flavor name.
+
+#### Nvidia (`N`)
+
+We show the most popular recent generations here. older one are of course possible as well.
+
+##### Ampere (`a`)
+
+One Streaming Multiprocessor on Ampere has 64 (A30, A100) or 128 Cuda Cores (A10, A40).
+
+GPUs without MIG (one SM has 128 Cude Cores and 4 Tensor Cores):
+
+| Nvidia GPU | Tensor C | Cuda Cores | SMs | VRAM      | SCS name piece |
+|------------|----------|------------|-----|-----------|----------------|
+|  A10       |  288     |  9216      |  72 | 24G GDDR6 | `GNa-72-24`    |
+|  A40       |  336     | 10752      |  84 | 48G GDDR6 | `GNa-84-48`    |
+
+GPUs with Multi-Instance-GPU (MIG), where GPUs can be partitioned and the partitions handed
+out as as pass-through PCIe devices to instances. One SM corresponds to 64 Cuda Cores and
+4 Tensor Cores.
+
+| Nvidia GPU | Fraction | Tensor C | Cuda Cores | SMs | VRAM      | SCS GPU name   |
+|------------|----------|----------|------------|-----|-----------|----------------|
+|  A30       | 1/1      | 224      | 3584       |  56 | 24G HBM2  | `GNa-56-24`    |
+|  A30       | 1/2      | 112      | 1792       |  28 | 12G HBM2  | `GNa-28-12`    |
+|  A30       | 1/4      |  56      |  896       |  14 |  6G HBM2  | `GNa-14-6`     |
+|  A30X      | 1/1      | 224      | 3584       |  56 | 24G HBM2e | `GNa-56h-24h`  |
+| A100       | 1/1      | 432      | 6912       | 108 | 80G HBM2e | `GNa-108h-80h` |
+| A100       | 1/2      | 216      | 3456       |  54 | 40G HBM2e | `GNa-54h-40h`  |
+| A100       | 1/4      | 108      | 1728       |  27 | 20G HBM2e | `GNa-27h-20h`  |
+| A100       | 1/7      |  60+     |  960+      |  15+| 10G HBM2e | `GNa-15h-10h`+ |
+| A100X      | 1/1      | 432      | 6912       | 108 | 80G HBM2e | `GNa-108-80h`  |
+
+[+] The precise numbers for the 1/7 MIG configurations are not known by the author of
+this document and need validation.
+
+##### Ada Lovelave (`l`)
+
+No MIG support, 128 Cuda Cores and 4 Tensor Cores per SM.
+
+| Nvidia GPU | Tensor C | Cuda Cores | SMs | VRAM      | SCS name piece |
+|------------|----------|------------|-----|-----------|----------------|
+|  L4        | 232      |  7424      |  58 | 24G GDDR6 | `GNl-58-24`    |
+| L40        | 568      | 18176      | 142 | 48G GDDR6 | `GNl-142-48`   |
+| L40G       | 568      | 18176      | 142 | 48G GDDR6 | `GNl-142h-48`  |
+| L40S       | 568      | 18176      | 142 | 48G GDDR6 | `GNl-142hh-48` |
```

Comment:
First, in the table columns, to be consistent it should be called "SCS GPU name" not "SCS name piece" I guess.
Second, for the L40S in the SCS GPU name column, there should be only 1 "h" behind the SMs, right?

Reply:
therefore because of the other comment, nevermind this.